core/src/main/java/com/graphhopper/routing/util/CarFlagEncoder.java

```diff
@@ -158,6 +158,9 @@ public int defineWayBits(int index, int shift) {
 
     protected double getSpeed(ReaderWay way) {
         String highwayValue = way.getTag("highway");
+        if (way.hasTag("motorroad", "yes")) {
```

Comment:
Should we override every highway? For example how about a motorway with motorroad (not sure if this combination is valid)?